src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Parameterize this service and add its implementation in a separate class.
  */
-public interface MachineService {
+public interface MachineService<T> {
```

Comment:
if we use only `T` type, we can parameterize this interface with any type of class (It can be Person, String, Object...) Is it valid behavior for `MachineService` interface?

Reply:
Changed to `<T extends Machine>` and made it more specific